Maxis/Controllers/LoginController.cs

```diff
@@ -0,0 +1,128 @@
+﻿using Maxis.Database;
+using Maxis.Interfaces;
+using Maxis.Repository;
+using System;
+using System.Collections.Generic;
+using System.DirectoryServices.AccountManagement;
+using System.Linq;
+using System.Web;
+using System.Web.Configuration;
+using System.Web.Mvc;
+using System.Web.Script.Serialization;
+using System.Web.Security;
+using Maxis.ViewModels;
+
+namespace Maxis.Controllers
+{
+    public class LoginController : Controller
+    {
+        private MaxisEntities db = new MaxisEntities();
```

Comment:
User DI and move all business logic & DB calls to its respective layer.. take help of nidhi if you have doubt.